### PR TITLE
[Term Entry]: PHP strtoupper()

### LINKED ISSUE #2436: [Term Entry] PHP String Functions strtoupper()
### Reason/inspiration (optional)

We would like a new term entry for [PHP](https://www.codecademy.com/resources/docs/php) about the `strtoupper()` function.

The entry should go in a new file in `docs/content/php/concepts/string-functions/terms/strtoupper/strtoupper.md`

The entry should 

- Have a syntax section illustrating the function with a ` ```pseudo ` code block. 
- An example showing the function in use with a ` ```php ` code block. 
- A runnable codebyte example in a ` ```codebyte/php` block. 

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by Jess-G95**
### Description

Added a new entry for the PHP function strtoupper(). The entry is in a new file in docs/content/php/concepts/string-functions/terms/strtoupper/strtoupper.md.

Closes #2436 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/php/concepts/string-functions/terms/strtoupper/strtoupper.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+## Syntax
+
+```pseudo
+strtoupper(string_to_convert);
+```
+
+The `string_to_convert` parameter is the string that will be converted to uppercase.
+
+## Codebyte
```
Comment (caupolicandiaz) [inline content/php/concepts/string-functions/terms/strtoupper/strtoupper.md:26]:
```suggestion
## Example
```

Comment (caupolicandiaz) [reply content/php/concepts/string-functions/terms/strtoupper/strtoupper.md:26]: generally we include one or more standard examples (depending on the entry) and zero or more codebytes (these are not always part of an entry)

File: content/php/concepts/string-functions/terms/strtoupper/strtoupper.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+The following code is runnable to show how `strtoupper()` alters the string.
```
Comment (caupolicandiaz) [inline content/php/concepts/string-functions/terms/strtoupper/strtoupper.md:28]:
```suggestion
The following code demonstrates the use of `strtoupper()` to alter a string.
```

File: content/php/concepts/string-functions/terms/strtoupper/strtoupper.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+```codebyte/php
```
Comment (caupolicandiaz) [inline content/php/concepts/string-functions/terms/strtoupper/strtoupper.md:30]:
```suggestion
```php
```

File: content/php/concepts/string-functions/terms/strtoupper/strtoupper.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+<?php
+echo strtoupper("hello, world!");
+?>
+```
```
Comment (caupolicandiaz) [inline content/php/concepts/string-functions/terms/strtoupper/strtoupper.md:34]: include a ```shell block with the output preceded by a sentence such as: "The code results in the following:"

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@Jess-G95 I've completed my initial review. Please update the entry, and tag me when you've finished making changes. Thanks

**Comment by Jess-G95** [comment]
@caupolicandiaz thanks, I have addressed your comments and made the changes.

**Review by SSwiniarski — APPROVED** [review]
@Jess-G95 Looks good to go.

**Comment by SSwiniarski** [comment]
@Jess-G95 Congrats on the new entry! Live link here: https://www.codecademy.com/resources/docs/php/string-functions/strtoupper

image

**Pull request merged**